helm/openwhisk/values.yaml

```diff
@@ -307,6 +307,42 @@ redis:
   persistence:
     size: 256Mi
 
+# User-events configuration
+user_events:
+  imageName: "adobeapiplatform/openwhisk-user-events" #TODO: Change to openwhisk/user-events after PR#4584 merge
+  imageTag: latest
+  imagePullPolicy: "IfNotPresent"
+  replicaCount: 1
+  restartPolicy: "Always"
+  port: 9095
+
+# Prometheus configuration
+prometheus:
+  imageName: "prom/prometheus"
+  imageTag: v2.10.0
+  imagePullPolicy: "IfNotPresent"
+  replicaCount: 1
+  restartPolicy: "Always"
+  port: 9090
+  persistence:
+    size: 1Gi
+  persistentVolume:
+    mountPath: /data
+
+# Grafana configuration
+grafana:
+  imageName: "grafana/grafana"
+  imageTag: "6.3.0"
+  imagePullPolicy: "IfNotPresent"
+  replicaCount: 1
+  restartPolicy: "Always"
+  port: 3000
+  adminPassword: "admin"
+  dashboards:
+  - https://raw.githubusercontent.com/adobe-apiplatform/openwhisk-user-events/master/compose/grafana/dashboards/openwhisk_events.json
+  - https://raw.githubusercontent.com/adobe-apiplatform/openwhisk-user-events/master/compose/grafana/dashboards/global-metrics.json
+  - https://raw.githubusercontent.com/adobe-apiplatform/openwhisk-user-events/master/compose/grafana/dashboards/top-namespaces.json
```

Comment:
is usage of adobe-apiplatform organization for the git content temporary staging or permanent?

Reply:
Temporary until [#4584](https://github.com/apache/openwhisk/pull/4584) is merged